x/distribution/autocli.go

```diff
@@ -0,0 +1,110 @@
+package distribution
+
+import (
+	"fmt"
+	"strings"
+
+	autocliv1 "cosmossdk.io/api/cosmos/autocli/v1"
+	distirbuitonv1beta1 "cosmossdk.io/api/cosmos/distribution/v1beta1"
+
+	"github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+)
+
+var (
+	FlagCommission       = "commission"
+	FlagMaxMessagesPerTx = "max-msgs"
+)
+
+// AutoCLIOptions implements the autocli.HasAutoCLIConfig interface.
+func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
+	exAccAddress, err := am.ac.BytesToString([]byte("A58856F0FD53BF058B4909A21AEC019107BA6A58856F0FD53BF058B4909A21AEC019107BA6"))
+	if err != nil {
+		panic(err)
+	}
+	valAddress, err := types.ValAddressFromHex("A58856F0FD53BF058B4909A21AEC019107BA6A58856F0FD53BF058B4909A21AEC019107BA6")
```

Comment:
This does not give me a good address: e.g `simd query distribution slashes cosmosvaloper15ky9du8a2wlstz6fpx3p4mqpjyrm56jcs4hsl4fm7pvtfyy6yxhvqxgs0wnqmanre8 0 100

Maybe let's just simplify the ux and in the example just say [validator address].